test/mocha/cli.js

```diff
@@ -650,4 +650,20 @@ describe("bin/uglifyjs", function () {
             done();
         });
     });
+    it("Should work with --rename", function(done) {
+        var command = uglifyjscmd + " test/input/rename/input.js --rename";
+        exec(command, function(err, stdout, stderr) {
+            if (err) throw err;
+            assert.strictEqual(stdout, "function f(a){return b(a);function b(c){return c}}\n");
+            done();
+        });
+    });
+    it("Should work with --no-rename", function(done) {
+        var command = uglifyjscmd + " test/input/rename/input.js -mc --no-rename";
+        exec(command, function(err, stdout, stderr) {
+            if (err) throw err;
+            assert.strictEqual(stdout, "function f(n){return function(n){return n}(n)}\n");
+            done();
+        });
+    });
```

Comment:
Need one more test to verify no option specified - `options.rename` is undefined.

Reply:
Done in https://github.com/mishoo/UglifyJS2/pull/2802/commits/1ac78ef700b4b57ee071f3c511ff11beb84fadd9